.github/scripts/patchwheel.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
We can add the content of https://github.com/arbor-sim/arbor/blob/master/LICENSE as a comment in this file and throw a one-liner into afdko's LICENSE noting that. Or we can add the content to afdko's LICENSE file like I did with the Antlr CMake stuff.